engine/src/image.cpp

```diff
@@ -413,6 +413,8 @@ Boolean MCImage::mdown(uint2 which)
 
 							return True;
 						}
+                    default:
```

Comment:
I think you need a `break;` for the preceding set of `case`s to prevent them falling through into the `default` case.